src/storage/rdb.cc

```diff
@@ -402,7 +405,7 @@ StatusOr<int> RDB::loadRdbType() {
 }
 
 StatusOr<RedisObjValue> RDB::loadRdbObject(int type, [[maybe_unused]] const std::string &key) {
-  if (type == RDBTypeString) {
+  if (type == RDBTypeString || type == RDBTypeBitmap) {
```

Comment:
The bitmap type is an internal represent type in Kvrocks, and it should be regarded as a string type. I think we can remove `RDBTypeBitmap`? also, the number `6` is now used by [RDB_TYPE_MODULE_PRE_GA](https://github.com/redis/redis/blob/unstable/src/rdb.h#L60C9-L60C31), reuse this number might cause the unexpected error while restoring rdb from Redis.

Reply:
Oh I see, thank you for the suggestion! Will do!